<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConnectionOptionsPage</class>
 <widget class="QWidget" name="ConnectionOptionsPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>550</height>
   </rect>
  </property>
  <property name="styleSheet">
   <string notr="true">QLabel, QCheckBox, QRadioButton {
    color: black;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="ProxyGroupBox">
     <property name="title">
      <string>Proxy Server</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QRadioButton" name="DirectConnectionRadioButton">
        <property name="text">
         <string>Connect to XenServer directly (no proxy)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="UseIERadioButton">
        <property name="text">
         <string>Use system proxy settings</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="UseProxyRadioButton">
        <property name="text">
         <string>Use the following proxy server:</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="ProxyAddressLabel">
          <property name="text">
           <string>Address:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="ProxyAddressTextBox">
          <property name="placeholderText">
           <string>proxy.example.com</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="ProxyPortLabel">
          <property name="text">
           <string>Port:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="ProxyPortTextBox">
          <property name="placeholderText">
           <string>80</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="BypassForServersCheckbox">
        <property name="text">
         <string>Bypass proxy server for XenServer connections</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="AuthenticationCheckBox">
        <property name="text">
         <string>Provide credentials for proxy authentication</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QFormLayout" name="formLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="ProxyUsernameLabel">
          <property name="text">
           <string>Username:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="ProxyUsernameTextBox"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="ProxyPasswordLabel">
          <property name="text">
           <string>Password:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="ProxyPasswordTextBox">
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="AuthMethodLabel">
        <property name="text">
         <string>Authentication method:</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QRadioButton" name="DigestRadioButton">
          <property name="text">
           <string>Digest</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="BasicRadioButton">
          <property name="text">
           <string>Basic</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="TimeoutGroupBox">
     <property name="title">
      <string>Connection Timeout</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="ConnectionTimeoutLabel">
        <property name="text">
         <string>Timeout (seconds):</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="ConnectionTimeoutSpinBox">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>300</number>
        </property>
        <property name="value">
         <number>20</number>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
